get-mods.sh

```diff
@@ -22,7 +22,9 @@ get_special () {
 				REF="$(get_tag)"
 				;;
 			*)
-				if [ -n "$BRANCH" ] && git -C dockerweb2-temp show -s --oneline "$BRANCH"; then
+				if [ "$BRANCH" = master ] && [[ "$2" == icinga-php/* ]]; then
+					REF=snapshot/nightly
```

Comment:
... and why not to just test for snapshot/nightly and fall back to master?

Reply:
Why would this be better in any way? If the script parameters say to use branch master, it will still use branch master unless on repositories where it's known that master is the wrong branch.